attach_file_to_run.py

```diff
@@ -0,0 +1,74 @@
+"""This script shows how to get the steps in a run and attach a file to
```

Comment:
also not a big deal, Multi line doc strings shouldn't be on the same level as the starting """

Reply:
Done!